package.json

```diff
@@ -27,6 +27,9 @@
     "q": "1.4.1",
     "rx": "4.1.0"
   },
+  "peerDependencies": {
+    "fh-mbaas-api": "7.0.0-3"
```

Comment:
@grdryn Good point, I don't think anything super important but I will update this now to just point at `latest-7` until it's time to merge this PR, then I'll set it to a specific version.